<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="Line" name="lineBottom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="lineTop">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="actionLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>86</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>86</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Action:</string>
      </property>
      <property name="buddy">
       <cstring>actionComboBox</cstring>
      </property>
     </widget>
     <widget class="QComboBox" name="actionComboBox">
      <property name="currentText">
       <string>Use this device (enable)</string>
      </property>
      <item>
       <property name="text">
        <string>Use this device (enable)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Do not user this device (disable)</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QToolButton" name="sourceToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="deviceClassLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Device Class:</string>
       </property>
       <property name="buddy">
        <cstring>deviceClassLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="deviceClassLineEdit">
       <property name="text">
        <string>Placeholder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="deviceTypeLabel">
       <property name="text">
        <string>Device Type:</string>
       </property>
       <property name="buddy">
        <cstring>deviceTypeLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="deviceTypeLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>deviceClassLineEdit</tabstop>
  <tabstop>sourceToolButton</tabstop>
  <tabstop>deviceTypeLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
